<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 154, 191);
background-color: rgb(253, 245, 223);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>700</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>750</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>USERNAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputUsername">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
            <property name="placeholderText">
             <string>example</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>PASSWORD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPassword">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
            <property name="placeholderText">
             <string>example.1234</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>CONFIRM PASSWORD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputConfirmPassword">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
            <property name="placeholderText">
             <string>example.1234</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>PHONE NUMBER</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPhone">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>11</number>
            </property>
            <property name="placeholderText">
             <string>05345678901</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>EMAÄ°L</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputEmail">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>25</number>
            </property>
            <property name="placeholderText">
             <string>email@email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>NAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputName">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
            <property name="placeholderText">
             <string>example</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>SURNAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputSurname">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
            <property name="placeholderText">
             <string>example</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>GENDER</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                <widget class="QRadioButton" name="btnRadioMale">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>MALE</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QRadioButton" name="btnRadioFemale">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>FEMALE</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>BIRTHDATE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="studentBirthdate">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>DEPARTMENT NAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputDepartmentName">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="maxLength">
             <number>6</number>
            </property>
            <property name="placeholderText">
             <string>COMP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>JOB STATUS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                <widget class="QRadioButton" name="btnRadioWorking">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Working</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QRadioButton" name="btnRadioNotWorking">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Not Working</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btnRegister">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 127);</string>
              </property>
              <property name="text">
               <string>REGISTER</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnGoBack">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 127);</string>
              </property>
              <property name="text">
               <string>GO BACK</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
